config/newrelic.yml.erb

```diff
@@ -0,0 +1,41 @@
+
+<% unless ENV['MAS_ENVIRONMENT'] == 'qa' %>
+  common: &default_settings
+    # Required license key associated with your New Relic account.
+    license_key: <%= ENV['NEWRELIC_API_KEY'] %>
+
+    # Your application name. Renaming here affects where data displays in New
+    # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+    app_name: Contento
+
+    # To disable the agent regardless of other settings, uncomment the following:
+    # agent_enabled: false
+
+    # Logging level for log/newrelic_agent.log
+    log_level: info
+
+
+  # Environment-specific settings are in this section.
+  # RAILS_ENV or RACK_ENV (as appropriate) is used to determine the environment.
+  # If your application has other named environments, configure them here.
+  development:
+    <<: *default_settings
+    app_name: Contento (Development)
+
+    # NOTE: There is substantial overhead when running in developer mode.
+    # Do not use for production or load testing.
+    developer_mode: true
+
+  test:
+    <<: *default_settings
+    # It doesn't make sense to report to New Relic from automated test runs.
+    monitor_mode: false
+
+  staging:
+    <<: *default_settings
+    app_name: Contento (Staging)
+
+  production:
+    <<: *default_settings
+    app_name: Contento (Production)
+<% end %>
```

Comment:
Ok so to check I've understood this right:

* No configuration is set for QA env
* Configuration is set for development, test, staging and production
* Staging and production are the same (except for the app_name)
* Test mode reports nothing to New Relic (I don't know the diff between this and QA env?)
* Development does report but also enables extra developer features

Am I on the right lines?